### PR TITLE
Refactored models from file to a directory

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
r @nancywong 

Also, @aharelick or @jondubin can you tell me if you think this is the right approach? Should we do it this way in flask-base?

File: app/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""
+These imports enable us to make all defined models members of the models
+module (as apposed to just their python files)
```
Comment (rileynwong) [inline app/models/__init__.py]: typo - opposed

Comment (sandlerben) [inline app/models/__init__.py]: :panda_face:

**Comment by maxmcc** [comment]
Jumping in here: I think this is a _much_ better structure than we've had in previous projects. We should definitely adopt this in flask-base.

**Comment by sandlerben** [comment]
Merged into flask-base instead

**Pull request closed without merging**